packages/marshal/src/helpers/copyArray.js

```diff
@@ -17,28 +17,23 @@ export const CopyArrayHelper = harden({
   styleName: 'copyArray',
 
   canBeValid: (candidate, check) =>
-    isArray(candidate) || check(false, X`Array expected: ${candidate}`),
+    isArray(candidate) ||
+    (!!check && check(false, X`Array expected: ${candidate}`)),
```

Comment:
How do you feel about dropping the `!!`?

Reply:
`canBeValid` is typed to return boolean, so dropping that upsets TypeScript:
```sh
$ tsc -p jsconfig.json
src/helpers/copyArray.js:16:14 - error TS2322: Type '{ styleName: "copyArray"; canBeValid: (candidate: any, check: Checker | undefined) => boolean | undefined; assertValid: (candidate: any, passStyleOfRecur: PassStyleOf) => void; }' is not assignable to type 'PassStyleHelper'.
  The types returned by 'canBeValid(...)' are incompatible between these types.
    Type 'boolean | undefined' is not assignable to type 'boolean'.
      Type 'undefined' is not assignable to type 'boolean'.

16 export const CopyArrayHelper = harden({
                ~~~~~~~~~~~~~~~


Found 1 error in src/helpers/copyArray.js:16
```